utilities/user_utilities.py

```diff
@@ -109,6 +112,18 @@ def get_all_user_events_count(group_ids, select_status, start=None, end=None):
                                         # "createdByGroupId": {"$in": group_ids},
                                         "eventStatus": {"$in": select_status}}))
 
+def get_all_device_status_pagination(skip, limit):
+    statusRpc = RpcClient()
```

Comment:
We donot need to create receiver every funtion call.
And can you rename RpcClient?